### DISCUSSION TITLE
Difference between Raw Value and Recognition

### DISCUSSION THREAD

**Discussion by amzrhh** [jomjol/AI-on-the-edge-device#950] [Q&A]
Hi Experts,
frequently it happens, that there is a difference between the Raw value and the values seen in the recognition window and the values in the recognition window are the good ones. 
See images attached.  
May be I made some wrong settings in the post procession chapter of the config.ini.
Do you have an explanation?
best regards,
August 

![01](https://user-images.githubusercontent.com/108123803/187026087-3d6a4234-d5e2-495a-9a9c-8f3013adfc36.JPG)
![02](https://user-images.githubusercontent.com/108123803/187026088-23f4a898-6a86-4765-9dd9-c7a32180b3b1.JPG)
![03](https://user-images.githubusercontent.com/108123803/187026090-9f651fb4-4990-4dd2-90bf-3899ed515d74.JPG)

**Comment by JensHoRi** [comment]
I believe it’s because of round. Could be fixed with new release and inter digits, otherwise the activation of max change helps.

**Comment by ghost** [comment]
See #942 
Which version do you use?

**Reply by JensHoRi** [reply]
The one from last week. master - v11.2.0 - 2022-08-28

**Reply by ghost** [reply]
Added a test case. Works with current rolling fine.

**Comment by nor500** [comment]
I am having similar problem with a regular rectangular shaped gas meter. Raw value shows 3249.459 on overview page. Recognition page shows corrects values of the reading 3249.69
![overview page](https://user-images.githubusercontent.com/5411152/190633038-2a4c32af-b888-4884-a0ed-55f4ad7e0f29.png)
![recognition page](https://user-images.githubusercontent.com/5411152/190633101-65d9f24d-b205-41ee-a290-af5018b78e56.png)

**Comment by JensHoRi** [comment]
That’s because an earlier reading gave another value. Just look how often this happens

**Comment by sohnvonoff** [comment]
Hi Developer,
I installed (not update) my three meters (gas, water, electric) with the new version 11.3 (master).
The gas meter runs perfectly. The electric and water meter shows the effects mentioned here.
water:
![grafik](https://user-images.githubusercontent.com/42845520/192348575-e6afcc23-cb85-40eb-b7ab-c44cd33ad5aa.png)
electric:
![grafik](https://user-images.githubusercontent.com/42845520/192349427-a707581c-c370-4d4e-bdb1-4f088bfd7b88.png)

The RAW value is always good. How does the algorithm arrive at the calculated values?
Attached are two log files.

[elektro.log](https://github.com/jomjol/AI-on-the-edge-device/files/9648721/elektro.log)
[Wasser.log](https://github.com/jomjol/AI-on-the-edge-device/files/9648770/Wasser.log)

best regards,

Bernd

**Comment by jomjol** [comment]
Please disable "CheckDigitIncreaseConsistency" in the expert settings. Know bug - to be corrected soon.

**Comment by 00Snider** [comment]
Hello to all,

i am dealing with the same problem. Raw Value: 331,... Recognition: 330,...

I am running the latest version with a water meter. Is there a fix that works?

Thanks for your help!
<img width="386" alt="Bildschirm­foto 2023-02-19 um 10 34 30" src="https://user-images.githubusercontent.com/76917507/219940212-8954ec70-03c6-4333-b8ad-7ab99dbdee57.png">

**Reply by ghost** [reply]
I don't know the pictures before, but I would say that it is a 330 and not a 331. So I don't see an issue here.

**Reply by ghost** [reply]
Ok, sorry see it. Raw means after post process. You can set the AnalogDigitalTransitionStart to 8.6

**Reply by 00Snider** [reply]
Thanks. I saw this setting but despite the explanation in the settings i don't understand what exactly is changed by this setting. Can you give me an example?

**Reply by ghost** [reply]
It's explained here https://jomjol.github.io/AI-on-the-edge-device-docs/Watermeter-specific-analog---digital-transition/
